src/openms/source/CHEMISTRY/DigestionEnzymeProtein.cpp

```diff
@@ -46,10 +46,10 @@ namespace OpenMS
     c_term_gain_(""),
     psi_id_(""),
     xtandem_id_(""),
-    comet_id_(),
-    crux_id_(),
+    comet_id_(-1),
```

Comment:
not that important but in the future, we should transform these into =default constructors and define the default initialization value of the variables in the header.  Less error prone...